librz/util/list.c

```diff
@@ -56,7 +56,7 @@ RZ_API void rz_list_init(RzList *list) {
 	list->sorted = false;
 }
 
-RZ_API int rz_list_length(const RzList *list) {
+RZ_API ut32 rz_list_length(const RzList *list) {
```

Comment:
Since you touches the `RzList` functions - could you please add doxygen to those functions that you change?

Reply:
I also wonder if it makes sense to separate these changes into the independent PR to get it merged faster.

Reply:
done: check https://github.com/rizinorg/rizin/pull/1463